packages/webawesome/src/internal/parse.ts

```diff
@@ -0,0 +1,7 @@
+/** Parses a space-delimited set of tokens and returns an array with all whitespace removed. */
+export function parseSpaceDelimitedTokens(input: string): string[] {
+  return input
+    .split(' ')
+    .map(token => token.trim())
+    .filter(token => token !== '');
+}
```

Comment:
```suggestion
  return input
     .trim()
     .split(/(\s|\r|\n|\f)+/)
```

^ this should actually make it so you dont need to map or filter. Not a big deal, but small optimization if you care 🤷